### ISSUE TITLE
Sorting "verletzte" is unintuitiv

### ISSUE THREAD

**Issue by sandzwerg** [codeformuenster/crashes-shiny#4]
What "Verletzte" does is really confusing:
Depending what I chose i get:

| chosen | outcome | heatmap /marker |  comment |
|-----------|--------|-------| -----| 
| leichtverletzt | many areas visible | heat |  |
| leichtverletzt | many  marker visible, some are outside the areas of before | marker |  |
| schwerverletzt | no area visible | heat |  |
| schwerverletzt | no marker visible | marker | Why doesn't I see the entries from schwer&leichtverletzt? | 
| schwerverletzt & leichtverletzt | no area visible | heat |  shouldn't I see all the leichtverletzt from befoire? |
| schwerverletzt & leichtverletzt | two marker visible | mark | Ok now two entries who have a schwer and leichtverletzt?!  | 
| Tote | map doesn't render | heat and/or mark | Even if there are no entries(really?) I'd expect a rendered map |
| Tote with leichtverletz/schwerverletzt or both | map doesn't render | heat and/or mark | I'd expect at least leicht & schwerverletzt to render |

In generall it's not clear to me if that is a "Verletzte" field is an supposed to be an intersection of the fields (i.e. markers have to fulfill all selections) or a join of the selected fields. Personally I'd expected it to be a join but right now I'm not sure what it is.

**Comment by silberzwiebel** [comment]
Thanks for your comprehensive bug report and sorry for the late reply. Somehow github didn't send me an e-mail notification.
Right now, the "Verletzte" option is connected with "AND", so if you select "Tote" and "leichtverletzt" you only get crashes where *both is true*. I implemented an OR conjunction the other day but this was confusing, too, out of reasons I can't remember. I'll do that again as it seems to be the more intuitive approach.

Regarding the heatmap / marker thing: Heatmap and markers are showing the same data but the heatmap might too sparse to be visible. I plan to make the heatmap adjustable (see #2), so that it even looks good with very little data points.

When you say "map doesn't render" do you mean that there is no background map at all? For me, if I choose a selection with 0 entries (true if you select "Tote" and leave everything else the same; try to remove "Rad" to see some entries), I can't select heatmap/marker -- but I still see the background OSM map.

**Comment by sandzwerg** [comment]
Another possiblity would be to chose if you want to AND or OR the "verletzte". I'd say there are usecases for both. Sometimes I want to see for example a heatmap over all kinds of injuries and sometimes I want to see where someone was killed or seriously injured. But I would also expect to see all crashes that involve "Schwerverletzte" when selecting "schwerverletzte" even if there were leichtverletzte or tote involved as well. 

As for the rendering: Yes the map doesn't render for me I only see a black background I've uploaded some screenshots as example: https://imgur.com/a/1qMdHAZ the first is the (not rendering) map for tote the second is the rendering one for schwerverletze (but appearing empty because of the heatmap). This is with FF 65 on a Fedora, on Chrome it seems to work I also tried that after youre remark see the third screenshot.
